app/src/main/java/org/gdg/frisbee/android/activity/ArrowActivity.java

```diff
@@ -116,7 +119,7 @@ protected String getTrackedViewName() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        Log.i(LOG_TAG, "onCreate");
+        Timber.i("onCreate");
```

Comment:
This could have been removed, esp. the info level is a bit too much. Looks like someone wanted to debug a certain lifecycle bug and didn't remove the line afterwards.